src/Resources/Resources/PrivateLinks/DeleteAzureResourceManagementPrivateLink.cs

```diff
@@ -0,0 +1,69 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Resources.PrivateLinks.Common;
+using Microsoft.Azure.Management.ResourceManager;
+
+namespace Microsoft.Azure.Commands.Resources.PrivateLinks
+{
+    /// <summary>
+    /// Remove-AzResourceManagementPrivateLink Cmdlet
+    /// </summary>
+    [Cmdlet(
+        VerbsCommon.Remove,
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ResourceManagementPrivateLink",
+        DefaultParameterSetName = Constants.ParameterSetNames.DeleteParameterSet,
+        SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class DeleteAzureResourceManagementPrivateLink : PrivateLinksCmdletBase
+    {
+        #region Cmdlet Parameters and Parameter Set Definitions
+
+        [Parameter(
+            ParameterSetName = Constants.ParameterSetNames.DeleteParameterSet,
+            Mandatory = true,
+            HelpMessage = Constants.HelpMessages.ResourceGroupName,
+            Position = 0)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; } = null;
+
+        [Parameter(
+            ParameterSetName = Constants.ParameterSetNames.DeleteParameterSet,
+            Mandatory = true,
+            HelpMessage = Constants.HelpMessages.PrivateLinkName,
+            Position = 1)]
+        [ValidateNotNullOrEmpty]
+        public string PrivateLinkName { get; set; }
+        #endregion
```

Comment:
- Is this delete operation long running operation? If so, please add -AsJob as parameter so that customer can run cmdlet in the background.
- Please add -Force to allow user to force run this cmdlet without confirmation.
- We also need Parameter PassThru so that customer can decide cmdlet whether returns true. 
 
Here is an example https://docs.microsoft.com/en-us/powershell/module/az.keyvault/remove-azkeyvault?view=azps-7.3.0#examples

Reply:
Delete operation is Synchronous.